### PR TITLE
Custom HTTP headers

### PR DESCRIPTION
**Pull request by tummychow**
Like so:

`p2-schedule --header X-Foo=bar --header X-Qux=baz manifest.yaml`

### CODE REVIEW

File: pkg/kp/client.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package kp
```
Comment (anthonybishopric) [inline pkg/kp/client.go]: This is a useful file but maybe it should be in a `pkg/util/netutil`?

Comment (anthonybishopric) [inline pkg/kp/client.go]: or just `pkg/netutil`

**Comment by anthonybishopric** [comment]
:+1: very useful for the various headers exposed by Consul like `X-Consul-LastContact`

File: pkg/kp/client.go (hunk 1 above)
Comment (tummychow) [inline pkg/kp/client.go]: I'll move it to `pkg/util/net`

**Pull request merged**